Fix formatting of installation commands

Chris-Johnston opened the pull request:
**Description:**

This change fixes the formatting of the generic linux host installation commands so that they all begin with `$`. This ensures that they are rendered correctly in the documentation.

Previously, the commands under `$ sudo -i` all began with `#` (to indicate that the shell was running as root). While this is correct, the markdown formatting thinks that they are comments, so they are rendered in a light gray color. I think that it's better that they are shown as commands and not comments.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html


<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9803"><img src="https://gitpod.io/api/apps/github/pbs/github.com/Chris-Johnston/home-assistant.io.git/f87c92c1dcffffd32c33aaef971cf49b29b41f98.svg" /></a>

Chris-Johnston committed:
Fix formatting of installation commands

This change fixes the formatting of the generic linux host
installation commands so that they all begin with `$`.
This ensures that they are rendered correctly in the documentation.

Previously, the commands under `$ sudo -i` all began with `#` (to
indicate that the shell was running as root). While this is correct,
the markdown formatting thinks that they are comments, so they
are rendered in a light gray color.

frenck reviewed (changes requested):
Actually, none of the commands produce output, so all preceding has to be removed.

See Markdown style guides rule 14:

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output

Chris-Johnston committed:
remove line start symbol from commads

none of the commands produce any output, so all of the symbols
should be removed

frenck reviewed (approved):
Thanks for adjusting this so quickly @Chris-Johnston! 👍
